src/contexts/SelectedUserContext.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { createContext, useState, useContext, useMemo, useEffect } from 'react';
+import { SignedInUserContext } from './SignedInUserContext';
+
+/**
+ * React Context for selecting users from PASS
+ *
+ * @name SelectedUserContext
+ * @memberof contexts
+ */
+
+export const SelectedUserContext = createContext({});
+
+export const SelectedUserContextProvider = ({ children }) => {
+  const { podUrl } = useContext(SignedInUserContext);
+  const [selectedUser, setSelectedUser] = useState({ podUrl });
+
+  const selectedUserMemo = useMemo(
+    () => ({
+      selectedUser,
+      selectUser: async (user) => setSelectedUser(user || { podUrl })
```

Comment:
Mentioned in a comment earlier, think we could include an entry for `person: "Personal Pod"` in the default `{ podUrl }` is `user` is not selected from SelectedUserContext.

Reply:
That sounds more like display text than data to me. Added it to the form page.